packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.Props.ts

```diff
@@ -0,0 +1,114 @@
+import { IButtonProps } from '../../Button';
+
+export interface ISwatchColorPickerProps {
+
+  /**
+   * the number of columns for the swatch color picker
+   */
+  columnCount: number;
+
+  /**
+   * The id for the swatch color picker
+   */
+  id?: string;
+
+  /**
+   * The shape of the color cells, defaults to circle
+   */
+  cellShape?: 'circle' | 'square';
+
+  /**
+   * The id of color cell that is currently selected
+   */
+  selectedId?: string;
+
+  /**
+   * Icon props for the swatch color picker. If given the swatch color picker
+   * will render inside of a menu
+   */
+  swatchColorPickerButtonProps?: IButtonProps;
+
+  /**
+   * Should the icon color be updated to align
+   * with the selected color? Default is false
+   */
+  updateButtonIconWithColor?: boolean;
+
+  /**
+   * The items to render the contents of the swatch color picker
+   */
+  swatchColorPickerItems: ISwatchColorPickerItemProps[];
+
+  /**
+   * Callback issued when the user changes the color
+   */
+  onColorChanged?: (color: string) => void;
```

Comment:
I was thinking the controlled component scenario, where the parent needs to specify the selectedId as new prop, it seems we won't be able to do that in the case the user clicks a "cell color"